addons/portal/views/address_templates.xml

```diff
@@ -428,7 +428,6 @@
                                                 id="use_delivery_as_billing"
                                                 name="use_delivery_as_billing"
                                                 class="form-check-input"
-                                                t-att-checked="use_delivery_as_billing"
                                             /> Add same for <t t-out="other_address_type"/> address
```

Comment:
`other_address_type` won't be translated here, you should duplicate the code with the two different labels